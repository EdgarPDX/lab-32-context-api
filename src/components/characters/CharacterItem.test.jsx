import React from 'react';
import { render } from '@testing-library/react';
import CharacterItem from './CharacterItem';


describe('Character component', () => {
  it('renders Character', () => {
    const { asFragment } = render(<CharacterItem
      name="test name"
      image="test image"
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
