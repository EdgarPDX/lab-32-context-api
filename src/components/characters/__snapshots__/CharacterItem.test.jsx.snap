// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character component renders Character 1`] = `
<DocumentFragment>
  <figure>
    <img
      alt="test name"
      src="test image"
    />
    <figcaption>
      test name
    </figcaption>
  </figure>
</DocumentFragment>
`;
